### DISCUSSION TITLE
三月更新 - 定

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#905] [通知]
## ✨ 重点

1. 支持 HDR 和 AV1 软解，以提高视频播放的稳定性
2. 支持从命令行启动，你可以通过参数命令让哔哩为你所用，具体使用方法参见仓库的 Wiki
3. 支持稍后再看的列表播放功能，一次看完囤积的视频

## 🚀 功能

- 支持连续播放稍后再看列表 @Richasy (#898)
- 支持自动播放关联视频 @Richasy (#897)
- 替换推荐视频数据源为平板源 @Richasy (#892)
- 超时自动隐藏历史记录提醒 @Richasy (#891)
- 扩大标题栏中返回按钮的可交互区域 @Richasy (#890)
- 支持鼠标中键及侧键返回 @Richasy (#888)
- 提供命令行启动方案 @Richasy (#885)
- 添加音量和倍速播放快捷键 @Richasy (#875)
- 支持播放速率快速切换 @Richasy (#874)
- 支持标签显示及跳转搜索 @Richasy (#873)
- 替换弹幕显隐图标 @Richasy (#871)
- 在番剧和国创页面添加我的追番按钮 @Richasy (#868)
- 添加全局播放速率设置 @Richasy (#867)
- 更新 FFmpeg 以支持 HDR 和 AV1 软解 @Richasy (#863)

## 🐛 Bug 修复

- 调整图片构建，界面优化 @Richasy (#904)
- 修复Xbox上返回按钮会直接退出应用的问题 @Richasy (#899)
- 修复密码登录问题 @Richasy (#889)
- 修复窄视图下关注直播间无法点击的问题 @Richasy (#886)
- 修复按下空格键有时无法播放/暂停的问题 @Richasy (#870)
- 修复播放 PGC 时没有正确跳转历史记录的问题 @Richasy (#869)
- 减少列表和图片控件嵌套以提高稳定性 @Richasy (#866)

## 🧰 管理与维护

- 更新 Issue 模板 @Richasy (#876)

<hr /><em>This discussion was created from the release <a href='https://github.com/Richasy/Bili.Uwp/releases/tag/v2.2203.5.0'>三月更新 - 定</a>.</em>

**Comment by xyl23333** [comment]
一天两更，勤快啊

**Comment by JohnsonRan** [comment]
好耶！

**Comment by SelfEnough** [comment]
联播有bug 退出当前播放的视频后他依旧会在后台播放关联视频 @Richasy

**Comment by sanbei101** [comment]
太牛逼了，竟然能做到这么频繁的更新程度
